draft-ietf-gnap-core-protocol.md

```diff
@@ -3363,39 +3371,44 @@ to either the access token's own key or, in the case of bearer tokens, the clien
 This method is indicated by `jwsd` in the
 `proof` field. A JWS {{RFC7515}} object is created as follows:
 
-The JOSE (JSON Object Signing and Encryption) header MUST contain the `kid` parameter of
-the key bound to this client instance for this request.  The `alg` parameter MUST be set
-to a value appropriate for the key identified by kid and MUST NOT be `none`.  The `b64`
-parameter MUST be set to `false` and the `crit` parameter MUST contain at least `b64`
-as specified in {{RFC7797}}.
+To protect the request, the JOSE header of the signature contains the following
+parameters:
 
-To protect the request, the JOSE header MUST contain the following
-additional parameters.
+kid (string)
+: The key identifier. RECOMMENDED. If the key is presented in JWK format, this
+  MUST be the value of the `kid` field of the key.
+
+alg (string)
+: The algorithm used to sign the request. REQUIRED. MUST be appropriate to the key presented. 
+  If the key is presented as a JWK, this MUST be equal to the `alg` parameter of the key. MUST NOT be `none`. 
 
 typ (string)
-: The type header, value ”gnap-binding+jwsd”.
+: The type header, value ”gnap-binding+jwsd”. REQUIRED
 
 htm (string)
-: The HTTP Method used to make this request, as an uppercase ASCII string.
+: The HTTP Method used to make this request, as an uppercase ASCII string. REQUIRED
 
-htu (string)
-: The HTTP URI used for this request, including all path and query components.
+uri (string)
+: The HTTP URI used for this request, including all path and query components and no fragment component. REQUIRED
 
 ts (integer)
```

Comment:
cat? created at

Reply:
If we were going to change this I'd rather just use `created` to be honest. The three-letter constraints of JWT don't have a lot of bearing here.

Reply:
changed to `created`